release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -75,6 +75,16 @@ Beginning with {product-title} {product-version}, you can specify either Network
 
 For more information, see xref:../installing/installing_aws/installing-aws-network-customizations.adoc[Installing a cluster on AWS with network customizations].
 
+[id="ocp-4-12-aws-local-zones-customization"]
+==== Extend worker nodes to the edge of AWS when installing into an existing VPC with Local Zone subnets.
+
+Now you can install {product-title} to an existing VPC with installer-provisioned infrastructure, extending the worker nodes to Local Zones subnets. The installer will provision worker nodes on the edge of the AWS network, with NoSchedule taints to be dedicated to the user applications.
```

Comment:
**suggestion:** Maybe use `With this update` or `With the new update` instead of `Now`.
For release notes, consider using 'With this update' rather than 'Now'.